library/src/scala/tasty/util/ShowSourceCode.scala

```diff
@@ -768,13 +800,35 @@ class ShowSourceCode[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty
     }
 
     def printDefinitionName(sym: Definition): Buffer = sym match {
-      case ValDef(name, _, _) => this += name
+      case ValDef(name, _, _) => this += name += ".type"
```

Comment:
why this change?

Reply:
I'm trying to figure out how to print `(x: @annot)`, the issue is that it gets expanded to something like `(x: x @annot)`. I was trying to workaround it by printing `(x: x.type @annot)` instead. I will now try to print it as `(x: @annot)` as it was originally.